lib/fog/google/requests/storage_json/head_object.rb

```diff
@@ -0,0 +1,54 @@
+module Fog
+  module Google
+    class StorageJSON
+      class Real
+        # Get headers for an object from Google Storage
+        #
+        # ==== Parameters
+        # * bucket_name<~String> - Name of bucket to read from
+        # * object_name<~String> - Name of object to read
+        # * options<~Hash>:
+        #   * 'If-Match'<~String> - Returns object only if its etag matches this value, otherwise returns 412 (Precondition Failed).
+        #   * 'If-Modified-Since'<~Time> - Returns object only if it has been modified since this time, otherwise returns 304 (Not Modified).
+        #   * 'If-None-Match'<~String> - Returns object only if its etag differs from this value, otherwise returns 304 (Not Modified)
+        #   * 'If-Unmodified-Since'<~Time> - Returns object only if it has not been modified since this time, otherwise returns 412 (Precodition Failed).
+        #   * 'Range'<~String> - Range of object to download
+        #   * 'versionId'<~String> - specify a particular version to retrieve
+        #
+        # ==== Returns
+        # * response<~Excon::Response>:
+        #   * body<~String> - Contents of object
+        #   * headers<~Hash>:
+        #     * 'Content-Length'<~String> - Size of object contents
+        #     * 'Content-Type'<~String> - MIME type of object
+        #     * 'ETag'<~String> - Etag of object
+        #     * 'Last-Modified'<~String> - Last modified timestamp for object
+        def head_object(bucket_name, object_name, options = {})
```

Comment:
Unused method argument - `bucket_name`. If it's necessary, use `_` or `_bucket_name` as an argument name to indicate that it won't be used. You can also write as `head_object(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `object_name`. If it's necessary, use `_` or `_object_name` as an argument name to indicate that it won't be used. You can also write as `head_object(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `head_object(*)` if you want the method to accept any arguments but don't care about them.